Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Wheel.mo

```diff
@@ -0,0 +1,284 @@
+within Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery;
+model Wheel "Sensible and latent air-to-air heat recovery wheels"
+  extends Buildings.Fluid.Interfaces.PartialFourPortInterface(
+        redeclare replaceable package Medium1 =
+        Modelica.Media.Interfaces.PartialCondensingGases,
+        redeclare replaceable package Medium2 =
+        Modelica.Media.Interfaces.PartialCondensingGases);
+  extends Buildings.Fluid.Interfaces.FourPortFlowResistanceParameters(
+        final computeFlowResistance1=true,
+        final computeFlowResistance2=true,
+        final from_dp1=false,
+        final from_dp2=false);
+
+  parameter Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.Types.RecoveryControlType controlType=
+    Buildings.Fluid.HeatExchangers.AirToAirHeatRecovery.Types.RecoveryControlType.Bypass
+    "Type of controller";
+  parameter Real P_nominal(unit="W") = 1000
+    "Power at design condition";
+  parameter Real k(final min=0) = 1
+    "Gain of controller";
+  parameter Modelica.Units.SI.Efficiency epsS_cool_nominal(final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsL_cool_nominal(final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsS_cool_partload(final max=1) = 0.75
```

Comment:
should the `0.75` be a hard-coded the value? I see the value is used in the comment. If not, then we should not put the `75%` in the comment.